components/ui/AppLogo.vue

```diff
@@ -79,9 +79,14 @@
   </svg>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Prop } from 'vue-property-decorator'
+<script setup lang="ts">
+interface Props {
+  color?: string
+}
+
+withDefaults(defineProps<Props>(), {
+  color: 'black'
+})
 
 let _uidCounter = 0
```

Comment:
As discussed offline, that's a good point you bring up.

Currently, it's quite hard to test given that the application doesn't work at the moment at this early stage of the refactor.

I added a TODO in https://github.com/Qiskit/qiskit.org/pull/2935/commits/f71ea9bd77dc72f458b16776736195b232a72542 so that we revisit this again when the refactor is at a stage where we can start testing the components.